### PR TITLE
Droth-2523 Visual upgrade to report table

### PR DESCRIPTION
**Pull request by RobertVisan**

### CODE REVIEW

File: UI/src/less/core/forms.less [hunk 1]
```diff
@@ -634,6 +634,10 @@ input[type="checkbox"] {
         display: none;
       }
     }
+    .multiSelectItems{
+      text-align: center;
+      margin-left: -10px
```
Comment (FilPinto) [inline UI/src/less/core/forms.less:639]: Maybe doesn't cause anything but add the ';' in this

margin-left: -10px;

**Review by marquesfm — APPROVED** [review]

File: UI/src/less/site/pages.less [hunk 2]
```diff
@@ -129,3 +145,13 @@
...
+
+.bigListPopUp {
+  overflow-y: auto;
+  height: auto;
+  max-height: 500px;
+}
+
+.clickableText {
+  cursor: pointer;
+}
```
Comment (FilPinto) [inline UI/src/less/site/pages.less:157]: You already have this one you can use:

.selectable{
  cursor: pointer;
}

File: UI/src/less/core/forms.less (hunk 1 above)
Comment (RobertVisan) [reply UI/src/less/core/forms.less:639]: Done

File: UI/src/less/site/pages.less (hunk 2 above)
Comment (RobertVisan) [reply UI/src/less/site/pages.less:157]: clickableText removed and selectable applied

**Pull request merged**